.github/workflows/main.yaml

```diff
@@ -66,13 +64,18 @@ jobs:
           conda-solver: libmamba
           python-version: "${{ matrix.python_version }}"
 
+      # Ensure compatible Poetry version after conda setup
+      - name: Install Poetry
+        run: python -m pip install "poetry>=1.2,<2.0"
+
       - name: Check Poetry lock
         run: poetry check
 
       - name: Install dependencies
         run: |
           poetry config virtualenvs.create false
-          poetry install
+          # Install both 'dev' and 'dicom' dependency groups for full CI coverage
```

Comment:
poetry install is a very readable command .. it is not necessary to add comments